apps/api-journeys/src/app/modules/block/button/button.resolver.ts

```diff
@@ -13,6 +14,17 @@ import { RoleGuard } from '../../../lib/roleGuard/roleGuard'
 @Resolver('ButtonBlock')
 export class ButtonBlockResolver {
   constructor(private readonly blockService: BlockService) {}
+
+  @ResolveField()
+  action(@Parent() block: ButtonBlock): Action | null {
+    return block.action != null
+      ? {
+          ...block.action,
+          parentBlockId: block.id
+        }
+      : null
```

Comment:
multi-line ternaries are really ugly. Can we do something a bit simpler like:
```suggestion
    if (block.action == null) return null
    
    return {
      ...block.action,
      parentBlockId: block.id
    }
```